### DISCUSSION TITLE
Website not loading

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23234] [Other Feature Feedback, Questions, & Ideas]
<p>I see my commit is verified but it won’t load:<br>
<a href="https://scratchimguploader.github.io/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://scratchimguploader.github.io/</a><br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/scratchimguploader/scratchimguploader.github.io" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/10137/181075684-989dfa74-4a0e-43a6-a533-85450e37d28d.png" class="thumbnail onebox-avatar" width="420" height="420">

<h3><a href="https://github.com/scratchimguploader/scratchimguploader.github.io" target="_blank" rel="noopener nofollow ugc">scratchimguploader/scratchimguploader.github.io</a></h3>

<p>Contribute to scratchimguploader/scratchimguploader.github.io development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by R4356th** [comment, marked as answer]
<p>It looks like you have not enabled GitHub Pages through the Settings tab on your repository yet. Go there (Settings) and scroll down to the GitHub Pages section, select the <code>main</code> branch as the source and click Save. And you should move this to <a class="hashtag" href="/c/github-pages/31">#<span>github-pages</span></a> for better assistance.</p>
